### DISCUSSION TITLE
i want to use copilot

### DISCUSSION THREAD

**Discussion by jxiaof** [community/community#19044] [Copilot Conversations]
free to me  - - -- - -- -- -

**Comment by ApidriuC** [comment]
Visit the next link and press "Start my free trial" : https://github.com/features/copilot/

![image](https://user-images.githubusercontent.com/78039311/174855758-80f70f31-8e48-4719-ace4-b23fa5b1627a.png)

**Comment by melvincwng** [comment]
Hi @jxiaof, you can start a free trial of GitHub Copilot using [this link](https://github.com/features/copilot/)

From their website, it would be a free 60-day free trial, then subsequently it would cost $10/month or $100/year per user.

Hope this clarifies.

Thanks :)

**Reply by melvincwng** [reply]
@jxiaof, 如果您觉得此回复有用，请“标记为答案” ('Mark as answer')，以便其他人知道此问题已得到解决。

谢谢。

**Comment by IvailoPetrov06** [comment]
You can get by clicking on and press start my free trial : https://github.com/features/copilot/
